### PR TITLE
Sadakat 10oct

### PR DESCRIPTION
**Pull request by sadakat-tech**

### CODE REVIEW

File: frontend/src/components/Simulation/SimulationDetails.js [hunk 1]
```diff
@@ -110,6 +110,7 @@ const SimulationDetails = () => {
   }
 
   const saveSimulation = async () => {
+    console.log(simulation)
```
Comment (romario-0) [inline frontend/src/components/Simulation/SimulationDetails.js]: Remove console logs

File: frontend/src/components/Simulation/SimulationDetails.js [hunk 2]
```diff
@@ -225,70 +226,83 @@ const SimulationDetails = () => {
 
   const createFieldElements = () => {
     return jobs.map((ele, idx) => (
-      <div key={`job_field_${idx}`}>
```
Comment (romario-0) [inline frontend/src/components/Simulation/SimulationDetails.js:228]: Key needs to be added in the topmost div

**Review by romario-0 — CHANGES_REQUESTED** [review]

File: frontend/src/components/Simulation/SimulationDetails.js (hunk 1 above)
Comment (sadakat-tech) [reply frontend/src/components/Simulation/SimulationDetails.js]: Removed Console

File: frontend/src/components/Simulation/SimulationDetails.js (hunk 2 above)
Comment (sadakat-tech) [reply frontend/src/components/Simulation/SimulationDetails.js:228]: changes done

**Pull request merged**